SETUP.md

```diff
@@ -0,0 +1 @@
+I wanna learn c++ with my bro
```

Comment:
Please add the following description "This is a mentorship program to teach C++ for everyone else."

Reply:
all was done